tools/buildHelper/buildHelper.psm1

```diff
@@ -94,17 +92,7 @@ function Get-ImageList
 
     if ($Channel -in 'community-stable', 'all')
     {
-        Get-ChildItem -Path $communityStablePath -Directory | Select-Object -ExpandProperty Name | Write-Output
-    }
-
-    if ($Channel -in 'community-servicing', 'all')
-    {
-        Get-ChildItem -Path $communityServicingPath -Directory -ErrorAction SilentlyContinue | Select-Object -ExpandProperty Name | Write-Output
-    }
-
-    if ($Channel -in 'community-preview', 'all')
-    {
-        Get-ChildItem -Path $communityPreviewPath -Directory -ErrorAction SilentlyContinue | Select-Object -ExpandProperty Name | Where-Object { $dockerFileNames -notcontains $_ } | Write-Output
+        Get-ChildItem -Path $communityStablePath -Directory | Select-Object -ExpandProperty Name | Where-Object { $dockerFileNames -notcontains "clearlinux" } | Write-Output
```

Comment:
I'd rather do this through metadata, but this is fine for now.  Please open an issue do undo this.

Reply:
Already in #192